lib/hooks/useFetchUserHighlights.ts

```diff
@@ -13,16 +13,15 @@ const useFetchUserHighlights = (username: string) => {
   const pageQuery = page ? `page=${page}` : "";
   const limitQuery = limit ? `&limit=${limit}` : "";
```

Comment:
If you deleted the usage of `pageQuery` & `limitQuery`, maybe you should delete their definitions as well, provided that we won't need them later on.

Reply:
The states as well.